phpseclib/Crypt/RSA.php

```diff
@@ -250,12 +250,20 @@ class RSA
     /**#@-*/
 
     /**
+     * Singleton instance
+     *
+     * @var \phpseclib\Crypt\RSA
+     * @access private
```

Comment:
Correct. Half the point is to hide it from phpdoc builds. See https://github.com/phpseclib/phpseclib/pull/813#commitcomment-13197976 .